### PR TITLE
Fix both utt2sid and utt2lid when removing long/short data

### PR DESCRIPTION
**Pull request by jonghwanhyeon**
Stage 3 of `tts.sh` removes long/short data. At this time, authors use `utils/fix_data_dir.sh` to remove the entry of the removed utterance as follows:

https://github.com/espnet/espnet/blob/e5d133c21800116b2fc5844859333ef83163d293/egs2/TEMPLATE/tts1/tts.sh#L500-L509

However, if there exist both `utt2sid` and `utt2lid`, `_fix_opts` are overwritten as `"--utt_extra_files utt2lid "` because authors used the assignment operator. So I tried to fix the above code as follows:

```bash
# fix_data_dir.sh leaves only utts which exist in all files
_fix_opts=""
if [ -e "${data_feats}/org/${dset}/utt2sid" ]; then
    _fix_opts+="--utt_extra_files utt2sid "
fi
if [ -e "${data_feats}/org/${dset}/utt2lid" ]; then
    _fix_opts+="--utt_extra_files utt2lid "
fi
# shellcheck disable=SC2086
utils/fix_data_dir.sh ${_fix_opts} "${data_feats}/${dset}"
```

However, after I tested it, it seems that `utils/fix_data_dir.sh` does not support multiple options. So, I suggest the following workaround in this PR becuase `utils/fix_data_dir.sh` is a part of `kaldi`:
```bash
# fix_data_dir.sh leaves only utts which exist in all files
if [ -e "${data_feats}/org/${dset}/utt2sid" ]; then
    utils/fix_data_dir.sh --utt_extra_files utt2sid "${data_feats}/${dset}"
fi
if [ -e "${data_feats}/org/${dset}/utt2lid" ]; then
    utils/fix_data_dir.sh --utt_extra_files utt2lid "${data_feats}/${dset}"
fi
```

### CODE REVIEW

File: egs2/TEMPLATE/tts1/tts.sh [hunk 1]
```diff
@@ -498,15 +498,12 @@ if ! "${skip_data_prep}"; then
                 awk ' { if( NF != 1 ) print $0; } ' >"${data_feats}/${dset}/text"
 
             # fix_data_dir.sh leaves only utts which exist in all files
-            _fix_opts=""
             if [ -e "${data_feats}/org/${dset}/utt2sid" ]; then
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/tts1/tts.sh:501]:
```suggestion
            _utt_extra_files=""
            if [ -e "${data_feats}/org/${dset}/utt2sid" ]; then
```

File: egs2/TEMPLATE/tts1/tts.sh [hunk 2]
```diff
@@ -498,15 +498,12 @@ if ! "${skip_data_prep}"; then
...
-                _fix_opts="--utt_extra_files utt2sid "
+                utils/fix_data_dir.sh --utt_extra_files utt2sid "${data_feats}/${dset}"
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/tts1/tts.sh:502]:
```suggestion
                _utt_extra_files+="utt2sid "
```

File: egs2/TEMPLATE/tts1/tts.sh [hunk 3]
```diff
@@ -498,15 +498,12 @@ if ! "${skip_data_prep}"; then
...
             fi
             if [ -e "${data_feats}/org/${dset}/utt2lid" ]; then
-                _fix_opts="--utt_extra_files utt2lid "
+                utils/fix_data_dir.sh --utt_extra_files utt2lid "${data_feats}/${dset}"
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/tts1/tts.sh:505]:
```suggestion
                _utt_extra_files+="utt2lid "
```

File: egs2/TEMPLATE/tts1/tts.sh [hunk 4]
```diff
@@ -498,15 +498,12 @@ if ! "${skip_data_prep}"; then
...
             fi
-            # shellcheck disable=SC2086
-            utils/fix_data_dir.sh ${_fix_opts} "${data_feats}/${dset}"
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/tts1/tts.sh:507]:
```suggestion
            # shellcheck disable=SC2086
            utils/fix_data_dir.sh --utt_extra_files "${_utt_extra_files}" "${data_feats}/${dset}"
```

**Review by kan-bayashi — CHANGES_REQUESTED** [review]
Thank you for your report and fixing.
In your case, the fixing is not performed when there is no utt2lid or utt2sid.
So I changed the strategy. Could you check my suggestion?

**Comment by jonghwanhyeon** [comment]
No problem! Thanks 😄

File: egs2/TEMPLATE/tts1/tts.sh [hunk 5]
```diff
@@ -498,16 +498,15 @@ if ! "${skip_data_prep}"; then
...
+                _utt_extra_files+="utt2sid "
             fi
             if [ -e "${data_feats}/org/${dset}/utt2lid" ]; then
-                _fix_opts="--utt_extra_files utt2lid "
+                _utt_extra_files+="utt2lid "
             fi
             # shellcheck disable=SC2086
-            utils/fix_data_dir.sh ${_fix_opts} "${data_feats}/${dset}"
-
+            utils/fix_data_dir.sh --utt_extra_files "${_utt_extra_files}" "${data_feats}/${dset}"
```
Comment (kan-bayashi) [inline egs2/TEMPLATE/tts1/tts.sh:509]:
```suggestion
            utils/fix_data_dir.sh --utt_extra_files "${_utt_extra_files}" "${data_feats}/${dset}"

```

**Review by kan-bayashi — APPROVED** [review]

**Comment by kan-bayashi** [comment]
Thank you for your kind contribution!

**Pull request merged**